client/src/components/shared/node/_utils/useDrag.js

```diff
@@ -51,7 +57,7 @@ const useDrag = (position, index, isEndpointNode) => {
       window.removeEventListener("mousemove", handleMouseMove);
       window.removeEventListener("mouseup", handleMouseUp);
     }
-  }, [state.isDragging, handleMouseMove]);
+  }, [state.isDragging, handleMouseMove, handleMouseUp]);
```

Comment:
Again, Im not exactly sure how this effects things. I don't think its necessarily bad.....I guess we will see.